### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
## CONTACT ME SECTION
- This project was a collaboration.
- It involved adding a contact form to the mobile version portfolio project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Mobile-Version
 - This is the second part of building my portfolio website
-- I will create sections within the website where i will include my portfolio projects.
+- I will create sections within the website where I will include my portfolio projects.
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please update your readme header following the [Microverse Readme template](https://github.com/microverseinc/readme-template) and descript the different steps necessary for another developer to clone and make use of the project.

File: style.css [hunk 2]
```diff
@@ -6,23 +6,25 @@
...
+  height: 735px;
   background: #fff;
   margin-top: 10px;
   border-bottom-left-radius: 100px;
   margin-bottom: 80px;
   background-image: url(./images/background.png);
   background-repeat: no-repeat;
   background-position-y: 100px;
+
+  /* flex: 1; */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

File: style.css [hunk 3]
```diff
@@ -482,3 +484,121 @@ button {
...
+  line-height: 52px;
+  color: #fff;
+}
+
+#text {
+  width: 327px;
+  height: 110px;
+  text-align: center;
+
+  /* margin-top: 30px; */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

File: style.css [hunk 4]
```diff
@@ -482,3 +484,121 @@ button {
...
+  flex-direction: column;
+  align-items: center;
+  margin-top: 53px;
+}
+
+.fields {
+  display: flex;
+  flex-direction: column;
+
+  /* margin: 0 auto; */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Kevo-hacker , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -482,3 +482,117 @@ button {
...
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 53px;
+}
+
+.fields {
+  display: flex;
```
Comment (tugsem) [inline style.css:533]: - [ ] [OPTIONAL] I highly suggest you to use d-flex class for all your flex containers. Doing that, your code will be applied best practices for DRY principle.

**Review by tugsem — APPROVED** [review]
Hi @Kevo-hacker , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights ⭐ 
✅  changes Implemented requested by the reviewer.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**